src/services/image_analysis_service.py

```diff
@@ -20,10 +20,12 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+from typing import List
+
 from src.classes.analysis_results import AnalysisResults
 from src.classes.video_data import VideoData
 
 
-def run_image_analysis(video_data: VideoData) -> AnalysisResults:
+def run_image_analysis(video_data: VideoData) -> List[AnalysisResults]:
```

Comment:
Could we add a docstring here so that we know the input/outputs/anything else for the function?
And is this function returning the frames individually, or the objects themselves?

Reply:
So this is one of the pieces very likely to change. It was setup early on when we were giving the project some structure. Based on the design we talked through this would essentially be the `AnalyzeImageService`. I just reused this piece for the time being to give an idea of where we're headed. Based on what we talked about my understanding is that we'll return a list of objects, each of which holds information about a single frame. Such as the processed image we plan to display in Streamlit and the modeling results.

Reply:
I added a placeholder docsting. Let me know how this all sounds.

Reply:
Thanks for adding it . I like this and we can always update it if necessary.